github/resource_organization_block_test.go

```diff
@@ -64,10 +72,14 @@ func testAccCheckOrganizationBlockExists(n string) resource.TestCheckFunc {
 		}
 
 		username := rs.Primary.ID
-		conn := testAccProvider.Meta().(*Owner).v3client
-		orgName := testAccProvider.Meta().(*Owner).name
+		meta, err := getTestMeta()
+		if err != nil {
+			return err
+		}
+		conn := meta.v3client
+		orgName := meta.name
 
-		blocked, _, err := conn.Organizations.IsBlocked(context.TODO(), orgName, username)
+		blocked, _, err := conn.Organizations.IsBlocked(context.Background(), orgName, username)
```

Comment:
question: would we get access to `t.Context()` if we changed the function definition to `func testAccCheckOrganizationBlockExists(ctx context.Context, n string)`?